### ISSUE TITLE
About the performance

### ISSUE THREAD

**Issue by SitongZhou** [airalcorn2/Recurrent-Convolutional-Neural-Network-Text-Classifier#23]
Hi,

Thank you for sharing the code. I ran your code using IMDB dataset, and the accuracy was 0.5. I wonder why this happened. Have you run the model on any dataset to test the performance? Thanks.

**Comment by airalcorn2** [comment]
Can you provide more details? Did you train the model with a validation set to avoid overfitting? What was the performance on the validation set? Did the model converge? Etc.
